### ISSUE TITLE
[UX feedback]:

### ISSUE THREAD

**Issue by Salonikansal2** [build-trust/ockam#6207]
### Observed behavior

>  prerequisites, such as the installation of Python and other dependencies, should be clearly mentioned at the beginning of the guide. It's essential for beginners to have a complete list of what's required before starting.

>If things get messy and you want to start over, there's a "RESET" part at the end.  But it would be helpful to have clear instructions on how to do this.

>Explaining the use of curl and Ockam Orchestrator lke this 
It's like getting a new app on your phone. First, you need to use a special code (like a secret password) to download it. The code here is that long line starting with "curl." This step installs a tool called Ockam Command Line.
and
Think of Ockam Orchestrator as a big organizer for your apps. It helps them work together smoothly.

>By explaining these points in simpler language, it should make the process easier to understand for beginners who might be new to Ockam.

### Steps to reproduce

changes in readme.

### Expected behavior

no confusion ,which can be acheived by using layman language

### Ockam Version

latest

**Comment by mrinalwadhwa** [comment]
@Salonikansal2 thank you, this is super helpful! Thank you spending time on it.

We would love to recognize the contribution in our next release and add you to our contributors list.

For that, could you please:

Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate new pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository.

The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

**Comment by Salonikansal2** [comment]
Glad you found that useful.
Thank u for adding me in the contributors list . I have followed the above
steps as you said please check.

**Comment by Salonikansal2** [comment]
can you label it as hacktoberfest as well

**Comment by mrinalwadhwa** [comment]
> We would love to recognize the contribution in our next release and add you to our contributors list.
>
> For that, could you please:
>
> Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate new pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository.
>
> The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.


@Salonikansal2 Thank you, I don't see an opened pull request from you to accept the Contributor License Agreement.

I see the one where you changed a message
https://github.com/build-trust/ockam/pulls/Salonikansal2

Could you check again please.
